client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -195,8 +199,13 @@ const Checkout = React.createClass( {
 			return selectedSiteSlug
 				? `/plans/${ selectedSiteSlug }/thank-you`
 				: '/checkout/thank-you/plans';
-		} else if ( cart.create_new_blog && cartItems.hasDomainRegistration( cart ) && ! cartItems.hasPlan( cart ) ) {
-			const domainName = cartItems.getDomainRegistrations( cart )[ 0 ].meta;
+		} else if ( cart.create_new_blog && ! cartItems.hasPlan( cart ) ) {
```

Comment:
I don't think the cart can have `create_new_blog` at the same time as it contains a plan. I could be wrong about that.